src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -1004,6 +1070,36 @@ describe('manage nsfs cli account flow', () => {
                 const res = await exec_manage_cli(type, ACTIONS.UPDATE, account_options);
                 expect(JSON.parse(res.stdout).error.code).toBe(ManageCLIError.InvalidAccountName.code);
             });
+
+            it('cli account add - cli update account with supplemental groups)', async function() {
```

Comment:
Now you're testing `account update`, please update the `it` title in this part.
Why do you need the last bracket?
```suggestion
            it('cli account update - cli update account with supplemental groups', async function() {
```